src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -23,6 +23,14 @@ void EnGinkoMan_Stamp(EnGinkoMan* this, GlobalContext* globalCtx);
 void EnGinkoMan_Dialogue(EnGinkoMan* this, GlobalContext* globalCtx);
 void EnGinkoMan_SwitchAnimation(EnGinkoMan* this, GlobalContext* globalCtx);
 
+extern FlexSkeletonHeader D_0600C240[];
+extern Gfx D_0600B1D8[];
+extern AnimationHeader D_060008C0[];
+extern AnimationHeader D_060043F0[];
+extern AnimationHeader D_06004A7C[];
+extern AnimationHeader D_06004F40[];
+extern AnimationHeader D_06000AC4[];
```

Comment:
Externed `FlexSkeletonHeaders` and `AnimationHeaders` are usually not arrays right?